pnr/openroad-gui/meta.yaml

```diff
@@ -0,0 +1,68 @@
+# Use `conda-build-prepare` before building for a better version string.
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v', '')|replace('-', '')|default('0.X'), GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH|default('gUNKNOWN')) %}
+{% set lemon_version = '1.3.1' %}
+{% set lemon_md5 = 'e89f887559113b68657eca67cf3329b5' %}
+package:
+  name: openroad-gui
+  version: {{ version }}
+
+source:
+  - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
+    git_rev: master
+    patches:
+      - odb-drt-disable-tests.patch
+      - sta-add-install-options.patch
+      - gui-find-qt-only-if-enabled.patch
+      - deactivate-tests-dst-odb.patch
+  - url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ lemon_version }}.tar.gz
+    md5: e89f887559113b68657eca67cf3329b5
+    folder: third_party/lemon
+build:
+  # number: 202202031935
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20220203_1935
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+  ignore_run_exports_from:
+    # header-only libraries
+    - fmt
+
+requirements:
+  build:
+    - make
+    - cmake >=3.19
+    - swig 4.0.2
+    - bison
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - libboost 1.73
+    - eigen 3.3
+    - spdlog 1.8.1
+    - readline
+    - tk
+    - zlib
+    - fmt
+    - libgomp
+    - qt
+    - mesalib
```

Comment:
I think we can use https://anaconda.org/main/mesa-libgl-devel-cos6-x86_64 as a `cdt` instead, see https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#:~:text=On%20Linux%2C%20using,by%20them%20automatically.

Reply:
I agree :-)